src/components/behandler/VelgBehandler.tsx

```diff
@@ -66,7 +66,7 @@ export const VelgBehandler = ({
         </Radio>
       ))}
       <Radio
-        value={undefined}
+        value=""
```

Comment:
Fikk en snodig bug her som en test fanget opp: `undefined` ble til verdien `"on"` når man valgte radioknappen, som gjorde at validering ikke feilet (fordi "on" ikke er falsy). Vi ønsker valideringsfeil når man har valgt radioknappen for "Søk etter behandler" uten å ha søkt opp behandler.